lib/util.js

```diff
@@ -0,0 +1,74 @@
+/*
+ * Copyright (c) 2018-2020 Digital Bazaar, Inc. All rights reserved.
+ */
+
+const DER_PRIVATE_KEY_PREFIX = Buffer.from(
```

Comment:
We should move the contents of this `util.js` file into the Node-only `lib/crypto.js` file. (Since it should be left out of the browser build).

Reply:
probably, I'm looking into one further optimization, I'm not sure sure we need to call on node `createPrivateKey` and `createPublicKey` in gen key as it appears all we need are the Buffers containing the key material. We do still call on those functions in sign and verify.

Reply:
@dmitrizagidulin addressed this here: https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/3/commits/2db9b78f875d45c9debe228af2ba0d7f6b049082 and https://github.com/digitalbazaar/ed25519-verification-key-2020/pull/3/commits/267da57447389a809150bcbd2507a0f77f33ee70